content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -115,24 +82,85 @@ The Istio control plane must be installed with virtual machine integration enabl
     $ istioctl install -f "${WORK_DIR}"/vmintegration.yaml
     {{< /text >}}
 
-## Create files to transfer to the virtual machine
+## Create Namespace and Setup policies for VM
+1. Create Namespace that will host the VM
+
+    {{< text bash >}}
+    $ kubectl create namespace $SERVICE_NAMESPACE
+    {{< /text >}}
+
+1. Create a serviceaccount for VM
+
+    {{< text bash >}}
+    $kubectl create serviceaccount $SERVICE_ACCOUNT -n $SERVICE_NAMESPACE 
+    {{< /text >}}
+
+1. Get the external ip or internal ip of the VM and store to `"${VM_IP}"`
+
+    {{< text bash >}}
+    $ VM_IP="<the external ip or internal ip of the VM>"
+    {{< /text >}}
+    
+1. Create a new service
```

Comment:
IMO we should not create a Service yet. The flow should be:

* Onboard a VM
* Connect it to the mesh

We are mixing the two, and its confusing to users.

You do NOT need to create a Service to add a VM. And you can use an existing one

Reply:
yes I think for user first time use I think point it out will take convenience to customer